README.md

```diff
@@ -0,0 +1,194 @@
+# OpenIddict, as an OpenID Connect Addict
+
+### What's Open Iddict ?
+
+OpenIddict library aims to provide a simple and easy out of the box solution to implement your OpenID Connect 
+server for ASP.NET 5.
+OpenIddict leverages the use of Identity (for user management) and Entityframework (as store provider).
+
+### What's Open ID Connect ?
+
+Open ID is a protocol that allows multiple clients to delegate the Authentication to one or multiple Authority servers.
+Facebook, Google, Twitter, are all well-known as OpenId Authority Servers.
+
+### Why should I want/need my own Open ID Connect Server?
+
+There are many scenarios on when this may be useful, 
+
+ 1. Having multiple applications with one central place for authentication.
+ 2. Having non-trusted clients (like JavaScript, Desktop or Mobile applications).
+ 3. Needs or wants to use single sign-on for multiple services.
+
+In general terms, you want to have a centralized authentication server/login and then any allowed clients
+(yours or not) can use the authentication server to handle identity and claims.
+
+--------------
+
+## Getting Started
+
+To use OpenIddict Server you need to include OpenIddict as a dependency in your project.json:
+```
+    "dependencies": {
+        "OpenIddict": "1.0.0-*"
+    },
+```
+
+In ConfigureServices there's a handy extension method for IdentityBuilder that you need to set up.
```

Comment:
nit: consider using backticks for types/methods/properties: ConfigureServices -> `ConfigureServices`